plugins/remmina/__init__.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+"""Search and start Remmina connections."""
+
+from albertv0 import *
+import os
+import subprocess
+from glob import glob
+from re import search, IGNORECASE
+import configparser
+
+__iid__ = "PythonInterface/v0.2"
+__prettyname__ = "Remmina"
+__version__ = "0.1"
+__trigger__ = "rm"
+__author__ = "Oğuzcan Küçükbayrak"
+__dependencies__ = ["remmina"]
+
+iconPath = "/usr/share/icons/hicolor/scalable/apps/org.remmina.Remmina.svg"
```

Comment:
What if the icon is not found there? Could you perhaps copy the icon under the same directory and find that instead?
I'm doing the same for all the other plugins of this repo.

Reply:
This icon is somehow not the same as the one used by albert app launcher anyways, I'll pack the correct one with the module.